More User Friendly, easier track creation, Tensorflow 2.X

LjAquinox opened the pull request:
What the title say

LjAquinox committed:
Update README.md

LjAquinox committed:
Add files via upload

LjAquinox committed:
Update Game.py

LjAquinox committed:
Update Globals.py

LjAquinox committed:
Update Drawer.py

LjAquinox committed:
Update main.py

LjAquinox committed:
Add files via upload

LjAquinox committed:
Delete images directory

LjAquinox committed:
Update README.md

LjAquinox commented:
Sorry I'm new to posting things on github if there is anything I've done wrong tell me I'll update things.

If you want to see the repo : https://github.com/LjAquinox/Car-QLearning

AlexandreCGMeier commented:
@Code-Bullet please accept the pull request. This works. Thanks, @LjAquinox!

JamesLear92 commented:
This is beyond broken. Even if you fix all of the incorrect import names (Globals rather than Global, Game rather than Games) etc, the game state is butchered. Before it had forward and reverse momentum game state, now it only has one, so the AI cannot ever go backward, key for learning tracks.

I have fixed all of this and will upload a correct version that works soon.

LjAquinox commented:
> This is beyond broken. Even if you fix all of the incorrect import names (Globals rather than Global, Game rather than Games) etc, the game state is butchered. Before it had forward and reverse momentum game state, now it only has one, so the AI cannot ever go backward, key for learning tracks.
> 
> I have fixed all of this and will upload a correct version that works soon.

I agree with the naming issues. 100%, tbh this was done without any consideration for anything. I believe I removed the backward state to speed up the learning process (it's been years; I don't remember but that's what I expect)

Have fun fixing the other issues. There are a ton. Good luck.
